examples/analytics/airflow-on-eks/main.tf

```diff
@@ -0,0 +1,542 @@
+  enable_airflow = true
+  airflow_helm_config = {
+    name             = local.airflow_name
+    chart            = local.airflow_name
+    repository       = "https://airflow.apache.org"
+    version          = "1.6.0"
+    namespace        = module.airflow_irsa.namespace
+    create_namespace = false
+    timeout          = 360
+    description      = "Apache Airflow v2 Helm chart deployment configuration"
+    values = [templatefile("${path.module}/values.yaml", {
+      # Airflow Postgres RDS Config
+      airflow_db_user = local.airflow_name
+      airflow_db_name = module.db.db_instance_name
+      airflow_db_host = element(split(":", module.db.db_instance_endpoint), 0)
+      # S3 bucket config for Logs
+      s3_bucket_name          = aws_s3_bucket.this.id
+      webserver_secret_name   = local.airflow_webserver_secret_name
+      airflow_service_account = local.airflow_service_account
+    })]
+
+    set_sensitive = [
+      {
+        name  = "data.metadataConnection.pass"
+        value = data.aws_secretsmanager_secret_version.postgres.secret_string
+      }
+    ]
+  }
+  tags = local.tags
+}
+
+#---------------------------------------------------------------
+# RDS Postgres Database for Apache Airflow Metadata
+#---------------------------------------------------------------
+module "db" {
+  source  = "terraform-aws-modules/rds/aws"
+  version = "~> 5.0"
+
+  identifier = local.airflow_name
+
+  engine               = "postgres"
+  engine_version       = "14.3"
+  family               = "postgres14"
+  major_engine_version = "14"
+  instance_class       = "db.m6i.xlarge"
+
+  storage_type      = "io1"
+  allocated_storage = 100
+  iops              = 3000
+
+  db_name  = local.airflow_name
+  username = local.airflow_name
+  password = sensitive(data.aws_secretsmanager_secret_version.postgres.secret_string)
+  port     = 5432
+
+  multi_az               = true
+  db_subnet_group_name   = module.vpc.database_subnet_group
+  vpc_security_group_ids = [module.security_group.security_group_id]
+
+  maintenance_window              = "Mon:00:00-Mon:03:00"
+  backup_window                   = "03:00-06:00"
+  enabled_cloudwatch_logs_exports = ["postgresql", "upgrade"]
+  create_cloudwatch_log_group     = true
+
+  backup_retention_period = 1
+  skip_final_snapshot     = true
+  deletion_protection     = false
+
+  performance_insights_enabled          = true
+  performance_insights_retention_period = 7
+  create_monitoring_role                = true
+  monitoring_interval                   = 60
+  monitoring_role_name                  = "airflow-metastore"
+  monitoring_role_use_name_prefix       = true
+  monitoring_role_description           = "Airflow Postgres Metastore for monitoring role"
+
+  parameters = [
+    {
+      name  = "autovacuum"
+      value = 1
+    },
+    {
+      name  = "client_encoding"
+      value = "utf8"
+    }
+  ]
+
+  tags = local.tags
+  db_option_group_tags = {
```

Comment:
I assume we could probably remove these two tags since they are not related to the outcome here - avoid any confusion for users